vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -143,6 +161,24 @@ NS_SWIFT_NAME(BraveAds)
 - (void)reportAdNotificationEvent:(NSString *)uuid
                         eventType:(BATAdNotificationEventType)eventType;
 
+/// Get inline content ad for the given dimensions
+- (void)inlineContentAdsWithDimensions:(NSString*)dimensions
+                            completion:(void (^)(BOOL success,
+                                                 NSString* uuid,
+                                                 NSString* creativeInstanceId,
+                                                 NSString* title,
+                                                 NSString* description,
+                                                 NSString* imageUrl,
+                                                 NSString* dimensions,
+                                                 NSString* ctaText,
+                                                 NSString* targetUrl))completion
```

Comment:
All of these should exist in a class, this is too many things to pass through arguments. Why is there no mojom type for this ad structure?

Reply:
https://github.com/brave/brave-browser/issues/13048 will make this change, but not prioritized as of yet due to other priorities

Reply:
Fixed